src/test/ui/alloc-error/alloc-error-handler-bad-signature-2.stderr

```diff
@@ -17,9 +17,6 @@ LL | |  }
    = note: struct `core::alloc::Layout` and struct `Layout` have similar names, but are actually distinct types
 note: struct `core::alloc::Layout` is defined in crate `core`
   --> $SRC_DIR/core/src/alloc/layout.rs:LL:COL
-   |
-LL | pub struct Layout {
-   | ^^^^^^^^^^^^^^^^^
```

Comment:
What happened here?

Reply:
this is defined in liballoc, so we stop rendering it

Reply:
Oh, right, hadn't seen the third commit modified `runtest` (not used to looking below the stderr files).